src/Nordea/Components/Range.elm

```diff
@@ -0,0 +1,153 @@
+module Nordea.Components.Range exposing
+    ( Config
+    , Range
+    , init
+    , view
+    , withShowInterval
+    )
+
+-- CONFIG
+
+import Css
+    exposing
+        ( Style
+        , backgroundColor
+        , borderRadius
+        , cursor
+        , displayFlex
+        , flexDirection
+        , height
+        , justifyContent
+        , margin2
+        , marginTop
+        , pct
+        , pointer
+        , property
+        , pseudoElement
+        , rem
+        , row
+        , spaceBetween
+        , transparent
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html, div, input, span)
+import Html.Styled.Attributes exposing (css, name, type_)
+import Html.Styled.Events exposing (onInput)
+import Nordea.Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Nordea.Themes as Themes
+
+
+type alias Config msg =
+    { value : Float
+    , min : Float
+    , max : Float
+    , step : Maybe Float
+    , onInput : String -> msg
```

Comment:
As the onInput event returns a string and the `string -> float` conversion may fail, I think we need a decoder to achieve this. Check out the decoder and event in Dropdown.elm

```suggestion
    , onInput : Float -> msg
```